src/Tabs.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: Callback,
+}
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <div className="App__wrapper">
+      <ul className="App__list">
+        {tabs.map(tab => (
+          <li key={tab.id}>
+            <button
+              type="button"
+              onClick={() => onTabSelected(tab.id)}
```

Comment:
1. Pass the `onTabSelected` callback to the `Tabs`
   **it should be called whenever the user selects another tab.
   (Don't call the callback if the tab was not changed)**